source/blog/2018-07-20-the-ember-times-issue-56.md

```diff
@@ -38,11 +38,11 @@ Would you like to use dynamic import() to lazy-load deps into your Ember app rig
 
 ## [SECTION TITLE](#section-url)
 
-
 ---
 
-## [SECTION TITLE](#section-url)
+## [RFC Roundup with @rwjblue](https://emberweekend.com/episodes/rfc-roundup-with-rwjblue)
 
+The Ember Weekend podcast is back on the mic! Want to hear more about some recent RFCs? @rwjblue discusses [RFC #276: Named Arguments Syntax (`{{@foo}}`)](https://github.com/emberjs/rfcs/blob/master/text/0276-named-args.md), [RFC #213: Custom Components API](https://github.com/emberjs/rfcs/pull/213), and [RFC #311: Introduce `<AngleBracketInvocationSyntax />`](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md) in [Ember Weekend's latest podcast](https://emberweekend.com/episodes/rfc-roundup-with-rwjblue). Entertaining and informative! TIL "splatributes". Subscribe to Ember Weekend [here](https://itunes.apple.com/us/podcast/ember-weekend/id981719021) on iTunes. 🎙
```

Comment:
Would it be better to not write `{{@foo}}` at all with Goodbits?

Reply:
Yes, good point, I think it makes it easier to not forget about removing it in goodbits and I think in this case you could also be able to convey the message by writing `@foo` on its own. What do you or @tylerturdenpants think?

Reply:
I prefer clarity over assumption. As for goodbits, I think we may need to add it the template to remember to fix it before submitting it.